### PR TITLE
Integrate Eslint with the frontend code.

### PR DESCRIPTION
**Pull request by RishabhJain2018**
#352 solved. @aka-jain Please review the PR.

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+module.exports = {
+    "extends": "angular",
+    "plugins": [
+        "angular"
+    ]
+};
```
Comment (aka-jain) [inline .eslintrc.js]: @RishabhJain2018 : please add a new line here.

Comment (RishabhJain2018) [inline .eslintrc.js]: okk sure.

**Comment by RishabhJain2018** [comment]
@aka-jain Please review the PR.

**Comment by aka-jain** [comment]
@deshraj : Merge this PR, but don't close the issue as I need to check for Eslint again.

**Pull request merged**